billing/invoice/service.go

```diff
@@ -177,6 +256,37 @@ func (s *Service) SyncWithProvider(ctx context.Context, customr customer.Custome
 	return nil
 }
 
+func (s *Service) upsert(ctx context.Context, customerID string,
+	existingInvoice *Invoice, stripeInvoice *stripe.Invoice) error {
+	if existingInvoice != nil {
+		// already present in our system, update it if needed
```

Comment:
If month 1's invoice was not paid, are we updating the invoice of month 1 and adding balance of month 2?

Reply:
Discussed offline